splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/ErrorHandler.java

```diff
@@ -0,0 +1,123 @@
+package com.redhat.console.notifications.splunkintegration;
+
+import java.io.IOException;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.LoggingLevel;
+import org.apache.camel.Processor;
+import org.apache.camel.builder.endpoint.EndpointRouteBuilder;
+import org.apache.camel.http.base.HttpOperationFailedException;
+import org.apache.http.ProtocolException;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+abstract class ErrorHandler extends EndpointRouteBuilder {
+
+  
+
+  @ConfigProperty(name = "integrations.component.name")
+  String COMPONENT_NAME;
+
+  public String LOGGER_NAME;
+  
+  public static final String RETURN_TYPE = "com.redhat.console.notifications.history";
+
+  @Override
+  public void configure() throws Exception {
+    LOGGER_NAME = "com.redhat.console.notification.toCamel." + COMPONENT_NAME;
+
+    getContext().getGlobalOptions().put(Exchange.LOG_EIP_NAME, LOGGER_NAME);
```

Comment:
This global setting is redefined second time. I think it would affect it globally within the Camel Context. We can have it defined once on BaseRoutes.

Reply:
Done! Keeping only in BaseRoutes class